lib/ProductOpener/Users.pm

```diff
@@ -615,19 +621,108 @@ sub check_edit_owner($user_ref, $errors_ref) {
 }
 
 
-sub init_user() {
 
-	my $user_id = undef ;
-	my $user_ref = undef;
-	my $org_ref = undef;
+=head2 open_user_session($user_ref, $request_ref)
+
+Open a session, store it in the user object, and return a cookie with the session id in the request object.
+
+=head3 Arguments
+
+=head4 User object $user_ref
+
+=head4 Request object $request_ref
+
+=head3 Return values
+
+The cookie is returned in $request_ref
+
+=cut
+
+sub open_user_session($user_ref, $request_ref) {
+
+	my $user_id = $user_ref->{'userid'};
+
+	# Maximum of sessions for a given user
+	my $max_session = 10 ;
```

Comment:
Is that for the number of sessions we store in `<user>.sto` ? 

Because at first read, I though it was the maximum number of time we use a cookie for a session (which does not make sense, but it's hard to guess as you read the code).

```suggestion
	# Maximum number sessions to store for a given user in sto file
	my $max_session = 10 ;
```